prusa/connect/printer/camera.py

```diff
@@ -57,7 +57,9 @@ def is_sendable(self) -> bool:
     def is_timelapse(self) -> bool:
         """Is this snapshot part of the layer change shots?"""
         required_attributes = [
-            self.camera_id,
+            self.camera_token,
+            # TODO: M3D-7: Identify why this is not available during saving
+            # self.camera_id,
```

Comment:
⬇️ LOW PRIORITY

Will (possibly) be fixed with [M3D-7](https://magdiel3.atlassian.net/browse/M3D-7)